apps/trading/client-pages/markets/markets-page.tsx

```diff
@@ -125,78 +166,355 @@ export const MarketsPage = () => {
-
-  const marketTabs: {
-    [key: string]: { id: string; name: string };
-  } = {
-    open: {
-      id: 'open-markets',
-      name: t('Open'),
-    },
-    proposed: {
-      id: 'proposed-markets',
-      name: t('Proposed'),
-    },
-    closed: {
-      id: 'closed-markets',
-      name: t('Closed'),
-    },
+
+  const {
+    marketTypes,
+    marketStates,
+    assets,
+    searchTerm,
+    setMarketTypes,
+    setMarketStates,
+    setAssets,
+    setSearchTerm,
+    reset,
+  } = useMarketPageFilters((state) => ({
+    marketTypes: state.marketTypes,
+    setMarketTypes: state.setMarketTypes,
+    marketStates: state.marketStates,
+    setMarketStates: state.setMarketStates,
+    assets: state.assets,
+    setAssets: state.setAssets,
+    searchTerm: state.searchTerm,
+    setSearchTerm: state.setSearchTerm,
+    reset: state.reset,
+  }));
+
+  const isMarketTypeSelected = (marketType: MarketType) =>
+    marketTypes.includes(marketType);
+
+  const marketTypeFilterOptions: Record<MarketType, string> = {
+    PERP: t('Perpetuals'),
+    FUTR: t('Futures'),
+    SPOT: t('Spot'),
   };
 
+  const marketStateFilterOptions: Record<MarketState, string> = {
+    OPEN: t('Open'),
+    CLOSED: t('Closed'),
+    PROPOSED: t('Proposed'),
+  };
+  let marketStateTrigger: string | undefined = undefined;
+  if (marketStates.length > 0) {
+    if (marketStates.length === 1) {
+      marketStateTrigger = marketStateFilterOptions[marketStates[0]];
+    }
+    if (marketStates.length > 1) {
+      marketStateTrigger = t('Status ({{count}})', {
+        count: marketStates.length,
+      });
+    }
+  }
+
+  const assetFilterOptions = orderBy(marketAssets, (a) => a.symbol, 'asc');
+  let assetTrigger: string | undefined = undefined;
+  if (assets.length > 0) {
+    if (assets.length === 1) {
+      assetTrigger = marketAssets?.find((a) => a.id === assets[0])?.symbol;
+    }
+    if (assets.length > 1) {
+      assetTrigger = t('Assets ({{count}})', { count: assets.length });
+    }
+  }
+
+  const filteredMarkets = filter(markets || [], {
+    marketTypes,
+    marketStates,
+    assets,
+    searchTerm,
+  });
+  const defaultMarkets = filter(markets || [], DEFAULT_FILTERS);
+
+  let filterSummary: ReactNode = undefined;
+  if (filteredMarkets.length != defaultMarkets.length) {
+    const diff = defaultMarkets.length - filteredMarkets.length;
+    filterSummary = (
+      <div className="p-2 text-center">
+        <Trans
+          i18nKey={
+            diff > 0
+              ? '{{count}} results excluded due to the applied filters. <0>Remove filters</0>.'
+              : '{{count}} results included due to the applied filters. <0>Remove filters</0>.'
+          }
+          values={{ count: Math.abs(diff) }}
+          components={[
+            <button
+              key="reset-filters"
+              className="underline"
+              onClick={() => {
+                reset();
+              }}
+            >
+              Remove filters
```

Comment:
Translation

Reply:
Not needed here, this text will be replaces by `<0>Remove filters</0>` from translations.